docs/debuginfod.md

```diff
@@ -0,0 +1,45 @@
+# debuginfod client
+
+In order to symbolize ingested profiles, Parca needs to have debug information
+for the binaries that are being profiled. Debug information
+typically comprises ELF object files, DWARF debug data, and source code
+required for symbolization. However, sometimes, application packages distributed
+by various Linux distros strip away debug information to minimize the size of
+the binaries. Thankfully, there are public vendor distributed servers which make distro
```

Comment:
"Thankfully, there are publicly accessible servers, distributing debug information for various Linux package managers and distributions."